package.json

```diff
@@ -24,13 +24,18 @@
     },
     "devDependencies": {
         "@types/bluebird": "^3.5.33",
+        "@types/mocha": "^10.0.0",
         "@types/node": "^12",
         "@types/request": "^2.48.8",
+        "chai": "^4.3.7",
+        "dotenv": "^16.0.3",
+        "mocha": "^10.1.0",
+        "ts-mocha": "^10.0.0",
```

Comment:
We'll need to use `ts-jest` here instead of `ts-mocha`, per Whispir's Greenfield Architecture guide.